packages/ui/src/components/Visualization/Canvas/FormV2/fields/AutoField.tsx

```diff
@@ -2,11 +2,25 @@ import { FunctionComponent, useContext } from 'react';
 import { FormComponentFactoryContext } from '../providers/FormComponentFactoryProvider';
 import { SchemaContext } from '../providers/SchemaProvider';
 import { FieldProps } from '../typings';
+import { isDefined } from '../../../../../utils';
+import { CanvasFormTabsContext } from '../../../../../providers/canvas-form-tabs.provider';
+import { useFieldValue } from '../hooks/field-value';
 
 export const AutoField: FunctionComponent<FieldProps> = ({ propName, required, onRemove }) => {
+  const { selectedTab } = useContext(CanvasFormTabsContext);
   const { schema } = useContext(SchemaContext);
+  const { value } = useFieldValue<object>(propName);
   const formComponentFactory = useContext(FormComponentFactoryContext);
 
+  const isFieldDefined = isDefined(value);
+  const isObjectTypefield = schema.type === 'object' || 'oneOf' in schema || 'anyOf' in schema;
+  const IsFieldRequired = !isDefined(required) || (isDefined(required) && required);
```

Comment:
@lordrip , I brought back this check as root Expression field were not loading with it fields on Required tab.
[Screencast From 2025-02-14 20-36-40.webm](https://github.com/user-attachments/assets/370e0d2c-2fcf-486f-826e-b424552687c9)

Reply:
I can rename it later, no worries :+1: